core/src/main/java/com/adobe/aem/commons/assetshare/configuration/impl/AssetDetails404Servlet.java

```diff
@@ -52,10 +52,7 @@
     protected final void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response)
             throws ServletException, IOException {
         // Control which requests enter this method via accepts(..) below.
-
-        response.setStatus(SlingHttpServletResponse.SC_NOT_FOUND);
-        response.getWriter().print("The path to a valid and accessible asset must be provided to the Asset Details page for it to display.");
-        response.getWriter().flush();
+        response.sendError(SlingHttpServletResponse.SC_NOT_FOUND);
```

Comment:
Think its helpful to pass that message as param #2? I assume most ppl override the error page at some level, but maybe for "debugging"? Or better to just leave blank?